src/PhpSpreadsheet/Writer/Xlsx.php

```diff
@@ -210,7 +210,13 @@ public function save($pFilename)
             $zip = new ZipArchive();
 
             if (file_exists($pFilename)) {
-                unlink($pFilename);
+                //Added by muhammad.begawala@gmail.com
+                //'@' will stop displaying "Resource Unavailable" error because of file is open some where.
+                //'unlink($pFilename) !== true' will check if file is deleted successfully.
+                //Throwing exception so that we can handle error easily instead of displaying to users.
+                if (@unlink($pFilename) !== true) {
```

Comment:
This would be a better fit for `\PhpOffice\PhpSpreadsheet\Shared\File` so that it can used somewhere else too

Reply:
You are right, thats good Idea, But somehow I can's pass travis check, I fixed coding style issues like spaces But still shows Failed with no error, Can you please advise me how to pass that so that I can pull changes.
Thanks @PowerKiKi

Reply:
See https://travis-ci.org/PHPOffice/PhpSpreadsheet/jobs/435169066#L540

Reply:
I have checked there already, No errors, Anyways but I have added new pull request as you mentioned to create function in `/Shared/File.php` and updated `/Writer/Xlsx.php` as required.
Please check and let me know, Hope this time it will pass travis test. @PowerKiKi